presto-main/src/main/java/com/facebook/presto/SystemSessionProperties.java

```diff
@@ -810,6 +811,11 @@ public SystemSessionProperties(
                         "Use top N row number optimization",
                         featuresConfig.isOptimizeTopNRowNumber(),
                         false),
+                booleanProperty(
+                        OPTIMIZE_CASE_EXPRESSION_PREDICATE,
+                        "Optimize case expression predicates",
+                        featuresConfig.isOptimizeCaseExpressionPredicate(),
+                        true),
```

Comment:
And we normally start by disabling the feature by default

Reply:
Done.